examples/drops/backend/src/methods/create_drop.ts

```diff
@@ -30,19 +30,19 @@ export const create_drop = async (client: DropsClient): Promise<void> => {
     description: 'Description',
     city: 'Buenos Aires',
     country: 'Argentina',
-    start_date: toPOAPdate(today),
-    end_date: toPOAPdate(oneMonthFromToday),
-    expiry_date: toPOAPdate(twoMonthsFromToday),
-    event_url: 'https://poap.xyz/',
-    virtual_event: true,
-    secret_code: '123456',
+    startDate: toPOAPdate(today),
+    endDate: toPOAPdate(oneMonthFromToday),
+    expiryDate: toPOAPdate(twoMonthsFromToday),
```

Comment:
Wouldn't be nice if the SDK supported JavaScript internal `Date` instead of just string?

Reply:
Moved to https://github.com/poap-xyz/poap.js/issues/65